operator/src/main/java/org/keycloak/operator/controllers/KeycloakDeployment.java

```diff
@@ -329,22 +330,22 @@ private void configureHostname(StatefulSet deployment) {
         var envVars =  kcContainer.getEnv();
         if (this.keycloakCR.getSpec().isHostnameDisabled()) {
             var disableStrictHostname = List.of(
-                new EnvVarBuilder()
-                        .withName("KC_HOSTNAME_STRICT")
-                        .withValue("false")
-                        .build(),
-                new EnvVarBuilder()
-                        .withName("KC_HOSTNAME_STRICT_BACKCHANNEL")
-                        .withValue("false")
-                        .build());
+                    new EnvVarBuilder()
+                            .withName("KC_HOSTNAME_STRICT")
+                            .withValue("false")
+                            .build(),
+                    new EnvVarBuilder()
+                            .withName("KC_HOSTNAME_STRICT_BACKCHANNEL")
+                            .withValue("false")
+                            .build());
```

Comment:
I believe this change is out-of-scope of this PR. Please revert if possible.

Reply:
Done! Thanks for your review and comment.